packages/airnode-node/src/evm/networks.ts

```diff
@@ -1,14 +1,18 @@
 import { ethers } from 'ethers';
+// Documentation link: https://docs.api3.org/airnode/v0.2/reference/airnode-addresses.html
```

Comment:
```suggestion
// Documentation link: https://docs.api3.org/airnode/latest/reference/airnode-addresses.html
```

Reply:
We also need to tackle this on the global scale - there are many references to 0.2 and 0.3

Reply:
I'll create an issue